api/src/main/resources/innsending-api.yml

```diff
@@ -1017,6 +1017,167 @@ paths:
+              - name: Nav-Call-Id
+                in: header
+                description: En ID som identifiserer kallkjeden som dette kallet er en del av.
+                required: false
+                schema:
+                  type: string
+
+            responses:
+              200:
+                description: Slettet søknad/ettersending
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/BodyStatusResponseDto"
+              401:
+                description: Uautorisert
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/RestErrorResponseDto"
+                    example:
+                      message: Autentisering feilet
+                      timestamp: 2023-12-21T09:31:40.549032434
+                      errorCode: errorCode.unauthorized
+              404:
+                description: Ingen ettersending funnet
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/RestErrorResponseDto"
+                    example:
+                      message: Ingen ettersending funnet
+                      timestamp: 2023-12-21T09:31:40.549032434
+                      errorCode: errorCode.resourceNotFound
+              500:
+                description: Feil i backend
+                content:
+                  application/json:
+                    schema:
+                      $ref: "#/components/schemas/RestErrorResponseDto"
+                    example:
+                      message: Noe gikk galt, prøv igjen senere
+                      timestamp: 2023-12-21T09:31:40.549032434
+                      errorCode: somethingFailedTryLater
+            tags:
+              - eksternoppgave
+
+  /ekstern/v1/oppgave/skjema/{skjemanr}:
+    get:
+      summary: Kall for å hente opprettede søknader/ettersendinger for bruker gitt skjemanummer fra eksterne applikasjoner
+      description: Kan filtrere på 'soknad' | 'ettersendelse'
+      operationId: oppgaveHentSoknaderForSkjemanr
+
+      parameters:
+        - name: skjemanr
+          in: path
+          description: Skjemaet som søknaden er basert på
+          required: true
+          schema:
+            type: string
+          style: simple
+          example: "NAV 55-00.60"
+        - name: soknadstyper
+          in: query
+          description: "Komma-separert liste som filtrerer hvilke type søknader som skal returnes. Feks: 'soknad,ettersendelse'"
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/SoknadType"
+        - name: Nav-Call-Id
+          in: header
+          description: En ID som identifiserer kallkjeden som dette kallet er en del av.
+          required: false
+          schema:
+            type: string
+
+      requestBody:
```

Comment:
skjemanummer også i body i stedet for path?

Reply:
Er flyttet